core-bundle/tests/EventListener/StoreRefererListenerTest.php

```diff
@@ -75,7 +75,7 @@ public function refererStoredOnKernelResponseProvider(): \Generator
             [
                 'newRefererId' => [
                     'last' => '',
-                    'current' => 'hi/I/am/your_current_referer.html',
+                    'current' => '/hi/I/am/your_current_referer.html',
```

Comment:
This change seems unrelated to the changes above. How is this related?

Reply:
Not directly related - but since #4851 our `StoreRefererListener` stores _absolute_ URLs instead of relative ones. However, the tests were not changed to reflect that.